static/js/components/VideoPlayer.js

```diff
@@ -395,6 +395,12 @@ class VideoPlayer extends React.Component<*, void> {
       this.checkYouTube()
     }
     this.updateSubtitles()
+
+    this.player.hotkeys({
+      volumeStep:                0.1,
+      seekStep:                  5,
+      enableModifiersForNumbers: false
+    })
```

Comment:
Might be better to call this function only when the player is ready, to make sure it's initialized:
```javascript
      function onPlayerReady() {
        this.enableTouchActivity()
        this.hotkeys({
          volumeStep:                0.1,
          seekStep:                  5,
          enableModifiersForNumbers: false
        })
```